Aspire Budgeting/Facilities/ObjectFactory.swift

```diff
@@ -30,4 +31,9 @@ class ObjectFactory {
   lazy var userManager: UserManager =  {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mohitathwani/aspirebudgeting/blob/c60f90ed59460d52ef9c97dbb14a1cc7a2b431bb/Aspire Budgeting/Facilities/ObjectFactory.swift#L31"></span>Opening braces should be preceded by a single space and on the same line as the declaration.<br />
<code>opening_brace</code> <code>ObjectFactory.swift:31</code></td>
    </tr>
  </tbody>
</table>